src/templates/LDCETemplate.ts

```diff
@@ -25,7 +25,7 @@ export function LDCETemplate(passport: { data: object }) {
           },
           {
             name: "Address",
-            value: get("applicant.address.singleLine"),
+            value: get("applicant.address.singleLine") || get("_address.single_line_address"),
```

Comment:
I was working with the [Digital Planning Data Schemas 2.0](https://docs.google.com/spreadsheets/d/1ePihRD37-2071Wq6t2Y7QtBt7juySWuVP6SAF6T-0vo/edit#gid=733673669) to find which variables to use. Is that out of date?
It'd be great if each flow could generate a passport schema to work from instead. I've not figured out how to go about producing something like that yet but I might give it a good if the documentation is a little stale.

Thanks for the advice on the domain logic here. It sounds like it's the wrong call to fallback to the _site_ address. I'll stick with no fallback and construct the address in the same way as the agent address.

Reply:
Google sheet can unfortunately fall out of date, yes. It doesn't have any formal connection or automations to the database.

For a small flow like https://editor.planx.uk/opensystemslab/about-the-applicant, the "Download the flow schema" link in the Editor debug panel should do exactly what you're talking about - it will print a CSV that includes a column with every passport variable this flow sets! 
![Screenshot from 2023-05-24 14-49-40](https://github.com/theopensystemslab/planx-document-templates/assets/5132349/7489e0da-59c4-4657-bddb-497928020dae)